shared/downloads/jeos-20-64.ini

```diff
@@ -1,6 +0,0 @@
-[jeos-20-64]
-title = JeOS 20 x86_64
-url = http://assets-avocadoproject.rhcloud.com/static/jeos-20-64.qcow2.7z
```

Comment:
note for the future: how it comes that the link does not work? Shouldn't we first merge this patch?...